test/services/limited-partnerships/limited.partnerships.spec.ts

```
@@ -162,21 +156,14 @@ describe("LimitedPartnershipsService", () => {
             const response = await service.patchLimitedPartnership(
                 mockValues.TRANSACTION_ID,
                 mockValues.SUBMISSION_ID,
-                {
-                    type: "email",
-                    data: {}
-                }
+                {}
```

We no longer have a type and I am not sure that an empty object should return a 400 error.
For the 400 I would use a badly formatted email in order to have something coherent

---

Replaced with a badly formatted email.